lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -95,7 +95,7 @@
         </progress-button>
 
         <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed && main.state.getFlashState().percentage != 100">
```

Comment:
Does the `main.state.getFlashState().speed` here also need changing to `main.state.getFlashState().speed != '0 B'`, similar to the change in `flash.js`?

Reply:
Yup! Done.